packages/strapi-plugin-i18n/constants/index.js

```diff
@@ -2,6 +2,12 @@
 
 const isoLocales = require('./iso-locales');
 
+const DEFAULT_LOCALE = {
+  name: 'English',
+  code: 'en',
```

Comment:
is en a valid code ? I think we should peak the en-US from iso-locales instead

Reply:
Yes it is :)
It's the locale specified in the US. May need to ask to @sam-pires 
Why do you prefer "en-US"?

Reply:
That's the way it has been specified back in the days. 
I don't have all the why behind that decision but from what I saw during the exploration phase in most cases users will use 1 locale by language(en, fr, it), and only a few numbers of interviewees have this specific level of localization(en-US, en-CA).
"en" seems to better fit our users' use cases than "en-US"

Reply:
If en exists as a single locale I'm good 👍  jsut thought it wasn't valid :)